projects/storefrontlib/src/cms-components/product/product-list/container/product-list-component.service.ts

```diff
@@ -30,9 +31,6 @@ import { ProductListRouteParams, SearchCriteria } from './product-list.model';
  */
 @Injectable({ providedIn: 'root' })
 export class ProductListComponentService {
-  // TODO: make it configurable
-  protected defaultPageSize = 10;
```

Comment:
Mention in the docs that this property was removed (also say how customer can configure it now) and add method-property migration.